AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -7786,6 +7786,7 @@ consent.faz.net#%#AG_onLoad(function(){if(window.location.href.includes("consent
 ! "Button I Accept"
 consent.falkirkherald.co.uk#%#AG_onLoad(function(){if(window.location.href.includes("consentUUID")){var g=new MutationObserver(function(){var b=document.querySelector('button[title="I Accept"]');b&&b.click();});g.observe(document,{childList:!0,subtree:!0});setTimeout(function(){g.disconnect()},1E4)}});
 !
+zeit.de#%#AG_onLoad(function() { var el=document.querySelector('button[title="EINVERSTANDEN"]'); if(el) el.click(); });
```

Comment:
Please add a check for current url.

zeit.de/zustimmung?url=

Also a little timeout before they click happens (or mutationobserver).

Reply:
@BlazDT what if we change the current rule?
```
consent-cdn.zeit.de#%#AG_onLoad(function(){if(!document.cookie.includes('euconsent-v2') || window.location.href.includes('consent-cdn.zeit.de')){var a=new MutationObserver(function(){var b=document.querySelector('button.message-button');b&&(a.disconnect(),b.click())});a.observe(document,{childList:!0,subtree:!0});setTimeout(function(){a.disconnect()},1E4)}});
```

Reply:
Even better. Search did not show the rule for me.

Reply:
@jellizaveta please find the current rule and replace it with https://github.com/AdguardTeam/AdguardFilters/pull/109752#discussion_r801565423